sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -1850,13 +1851,15 @@ def create_pipeline_spec(
     pipeline: pipeline_context.Pipeline,
     component_spec: structures.ComponentSpec,
     pipeline_outputs: Optional[Any] = None,
+    pipeline_config: pipeline_config.PipelineConfig = None,
```

Comment:
YAPF (Python linter) check is failing for this file. Running this locally should resolve it:
`yapf -i sdk/python/kfp/dsl/pipeline_context.py sdk/python/kfp/dsl/pipeline_config.py sdk/python/kfp/compiler/pipeline_spec_builder.py`